index.html

```diff
@@ -1,164 +1,154 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Document</title>
-  
-  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
-  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
 
- <style>
-   .nav{float:right;
-   font-size: 30px;
-  }
-  main header{text-align: center;
-  height:100%;
-  width:100%;
-  background-color: skyblue;
-  padding-top:100px ;
-  }
-  .col-sm-3{
-      margin-right: 10px;
-  }
-  img{
-    width:450px;
-    height:200px;
-    padding: 20px;
-    margin: 20px;
+    <link
+      rel="stylesheet"
+      href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
 
+    <style>
+      .nav {
+        float: right;
+        font-size: 30px;
+      }
+      main header {
+        text-align: center;
+        height: 100%;
+        width: 100%;
+        background-color: skyblue;
+        padding-top: 100px;
+      }
+      .col-sm-3 {
+        margin-right: 10px;
+      }
+      img {
+        width: 450px;
+        height: 200px;
+        padding: 20px;
+        margin: 20px;
+      }
+      .pad {
+        padding-left: 10px;
+      }
+      .hr {
+        border-top: 5px;
+        border-color: royalblue;
+      }
+      .height {
+        padding-top: 10px;
+        margin-bottom: 20px;
+        height: 30px;
+      }
 
-  
-  }
-  .pad{
-    padding-left:10px  ;
-
-  }
-  .hr{
-      border-top: 5px ;
-      border-color: royalblue;
-     
-     
-  }
-  .height{
-    padding-top:10px ;
-    margin-bottom:20px ;
-    height:30px;
-
-  }
-
-  #welcome-section{
-    height:100vh;
-  }
-  @media screen and (min-width: 480px) {
-  p{
-    background-color: lightgreen;
-  }
-}
-
-
-  
- </style>
-<script>
-
-
-</script>
```

Comment:
I think script tag should be at the end of document. to increase reload time.  Kindly do not leave. empty code in production